website/client/components/header/menu.vue

```diff
@@ -149,7 +169,35 @@ div
           border-bottom: #6133b4 solid 1px;
         }
 
+        .chevron {
+          width: 20%;
+          height: 42px;
+          position: absolute;
+          right: 0;
+          top: 0;
+          display: block;
+        }
+
+        .chevron-icon-down {
+          width: 14px;
+          top: 11px;
+          right: 12px;
+          position: absolute;
+          display: block;
+          transition: transform 0.25s ease;
+     -moz-transition: transform 0.25s ease;
```

Comment:
can you remove these browser prefixes? We have a plugin that automatically adds them if needed :) Same below with the `rotate` rule